jobs/JREGIONAL_RUN_POST

```diff
@@ -47,7 +47,13 @@
 #-----------------------------------------------------------------------
 #
 . $USHdir/source_util_funcs.sh
-for sect in user nco platform workflow task_run_post ; do
+sections=(
+  user
+  nco
+  workflow
+  task_run_post
```

Comment:
Do you want to remove `task_run_post` since it is being called in a loop with `source_yaml`. The name of this file being `*_RUN_POST` makes me question if it should stay in, but based on the document you wrote possibly not. 

If it should stay, besides the naming of the file as a clue, why in this loop and not the rest that call `source_yaml`?

Reply:
I should probably go ahead and remove this file and its scripts/ counterpart.